subreddit.py

```diff
@@ -15,12 +15,17 @@
                      user_agent=config.user_agent_new)
 
 subreddits_old_account = list(old_account.user.subreddits(limit=None))
+
+
 print(f'Old account subscribed to {len(subreddits_old_account)} subreddits.\n\n')
 
+
 subreddits_new_account = list(new_account.user.subreddits(limit=None))
+print(f'New account subscribed to {len(subreddits_new_account)} subreddits.\n\n')
 
 for counter, sub in enumerate(subreddits_old_account):
-    new_account.subreddit(sub.display_name).subscribe()
```

Comment:
Thanks for submitting a fix. However, you removed the key line that subscribes the new account to the old subreddit. If you add it below the if statement, I'll be happy to accept it

Reply:
Hey. Looks like you missed this comment. Without this line, new account won't be able to subscribe to new accounts